Electron/AMAI-release/install.js

```diff
@@ -207,6 +207,26 @@ const installOnDirectory = async () => {
             process.send(f2AddToMPQ.error.message)
               : process.send(installCommander ? `Installing commander ${file}` : `Installing VS Vanilla AI commander ${file}`);
 
+        } else {
+          const f2AddToMPQ =  spawnSync(
+            `MPQEditor.exe`,
```

Comment:
This is dangerous, as uninstalling all the time could break maps that never installed the commander but do have a custom blizzard.j.

Reply:
There are a few buttons, it may be necessary to readjust the layout